AppCenter/AppCenter/MSAppCenter.h

```diff
@@ -31,6 +38,14 @@
  */
 + (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services;
 
+/**
+ * Configure the SDK with an array of services to start.
```

Comment:
Should we mention that it can start without app secret explicitly?

Reply:
I'd point out that a `configure`-call needs to be balanced with a `start` call.